### ISSUE TITLE
In the messaging area header, user and ad-hoc group chat names are ellipsed for no reason

### ISSUE THREAD

**Issue by imagiMary** [status-im/status-desktop#9693]
Version: 0.9
OS: Windows 10

In the messaging area header, there are various scenarios when user and ad-hoc group chat names are incorrectly ellipsed (e.g. “Mary_” is sometimes displayed as “Ma…” or “Mar…”) in spite of empty space being available to the right of the username.

_*Note: This bug is better demonstrated in the screencast below_

***

## Scenario 1

<ins>Steps to reproduce:</ins>

1. Navigate to Messages

2. Click on any ad-hoc group chat. _*Note: the username in the secondary nav is being correctly displayed._

<ins>What currently incorrectly happens</ins>

3. The ad-hoc group chat title in the messaging area header is incorrectly ellipsed e.g “Mary_&MrTesting” is displayed as “Mary_&MrTest…”

<ins>What is supposed to happen</ins>

3. The ad-hoc group chat title in the messaging area header should be displayed in it’s non-ellipsed form (because it doesn’t need to be ellipsed) as “Mary_&MrTesting”

## Scenario 2

<ins>Steps to reproduce:</ins>

1. Navigate to Messaging

2. Click on any username to open the 1on1 chat with that user. _*Note: the username in the secondary nav is being correctly displayed._

3. Click on a different user to open the 1on1 chat with them

<ins>What currently incorrectly happens</ins>

4. The username in the messaging area header is incorrectly ellipsed e.g “Mary_” is sometimes displayed as “Ma…”

<ins>What is supposed to happen</ins>

4. The username in the messaging area header should be displayed in it’s non-ellipsed form (because it doesn’t need to be ellipsed) as “Mary_”

### _Continuing scenario 2…_

5. Click on any ad-hoc group chat in the secondary nav to open it

6. Click back on the step 4. user 1on1 chat who's username was ellipsed

7. Notice how their username in the messaging area header is now correctly displayed in its non-ellipsed form e.g. “Mary_” is correctly displayed as “Mary_”  (!!!)

<ins>Other information</ins>

Community Channel usernames are also getting ellipsed in the messaging area header.

https://user-images.githubusercontent.com/124154641/222144383-90fff329-60b5-42d7-852e-903a7d8525c5.mp4

&nbsp;
### tldr; of what is supposed to happen

Usernames and ad-hoc group chat titles in the messaging area header should never be ellipsed if there is sufficient space to display the full username/ad-hoc group chat title.  As usernames and ad-hoc group chat titles are limited to a max of 24 characters, there should almost always be sufficient space for them to be displayed without being ellipsed.

**Comment by caybro** [comment]
Already fixed in master via https://github.com/status-im/status-desktop/pull/9602
